### PR TITLE
Who Are We section initial PR

### PR DESCRIPTION
**Pull request by selina-kim**
- created component for Who Are We
- copied basic structure and text from prototype design in figma
- closes TECH-21, TECH-40, and TECH-41

### CODE REVIEW

File: src/components/WhoAreWe.astro [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<div class="mx-auto max-w-lg">
+  <h1 class="text-6xl">Who Are We</h1>
+
+  <div class="flex flex-wrap p-4 gap-4">
+    <p class="text-sm">
```
Comment (err53) [inline src/components/WhoAreWe.astro]: Should be text-xl according to the design (20 pt)

https://tailwindcss.com/docs/font-size

File: src/components/WhoAreWe.astro [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<div class="mx-auto max-w-lg">
```
Comment (err53) [inline src/components/WhoAreWe.astro]: Take a look at tailwind's columns specifiers, I think we can use either a `md` or `lg` breakpoint to switch to 2 columns. (Also I think max-w-lg is unnecessary after we do that?)

https://tailwindcss.com/docs/columns

**Review by err53 — CHANGES_REQUESTED** [review]

File: src/components/WhoAreWe.astro (hunk 2 above)
Comment (err53) [reply src/components/WhoAreWe.astro]: Ok let's do the breakpoint stuff in a future change

File: src/components/WhoAreWe.astro (hunk 1 above)
Comment (err53) [reply src/components/WhoAreWe.astro]: @selina-kim Can you get this changed real quick? Thanks.

File: src/components/WhoAreWe.astro [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  <h1 class="text-6xl">Who Are We</h1>
```
Comment (err53) [inline src/components/WhoAreWe.astro]: Also please update the font by adding `font-display` to the class, and add `text-fill-transparent text-stroke-white text-stroke-2` where needed as per the design document

File: src/components/WhoAreWe.astro (hunk 1 above)
Comment (selina-kim) [reply src/components/WhoAreWe.astro]: fixed

Comment (selina-kim) [reply src/components/WhoAreWe.astro]: fixed

File: src/components/WhoAreWe.astro (hunk 3 above)
Comment (selina-kim) [reply src/components/WhoAreWe.astro]: Fixed. Also noticed that the figma wireframe design has 48px (text-5xl) for the header so I changed it to that. Should I have left it as text-6xl?

**Review by err53 — APPROVED** [review]
good for a first pass

**Comment by err53** [comment]
LGTM, @selina-kim feel free to merge when you get a chance

**Pull request merged**